torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -126,10 +126,81 @@ bool AliasDb::isContainerType(const TypePtr& type) const {
 
 AliasDb::~AliasDb() = default;
 
+// Structure that keeps tracks of all writes at a high level. When alias
+// analysis is done, this will be used to construct a more efficient WriteIndex.
+struct AliasDb::WriteRegistry {
```

Comment:
Maybe add comment somewhere that this exists before there is a value -> element mapping